build.rs

```diff
@@ -42,21 +67,38 @@ fn main() {
             .rustified_enum("nng_pipe_ev")
             .rustified_enum("nng_sockaddr_family")
             .rustified_enum("nng_zt_status")
+            .rustified_enum("nng_tls_mode")
+            .rustified_enum("nng_tls_auth_mode")
+            .rustified_enum("nng_http_status")
```

Comment:
Both `nng_errno_enum` and `nng_flag_enum` should be integers, not Rust enums. The flags are meant to be used as bit flags and the Error enum has two variants (`NNG_ESYSERR` and `NNG_ETRANERR`) that will lose information.

Reply:
Missed that, nice catch.